src/quo2/components/buttons/button/style.cljs

```diff
@@ -85,14 +85,14 @@
                                   56 0
                                   40 (if border-color 8 9)
                                   32 (if border-color 4 5)
-                                  24 (if border-color 0 1)
+                                  24 0
                                   (if border-color 8 9)))
           :padding-bottom     (when-not (or icon-only? icon-left icon-right)
                                 (case size
                                   56 0
                                   40 9
                                   32 5
-                                  24 4
+                                  24 0
```

Comment:
@J-Son89 I think these paddings need to be revised. They seem redundant to me as height is already given.

Reply:
I don't believe so, Im pretty sure I added these to get the text alignment correct in the outline variant.

Reply:
I tried removing padding-top, padding-bottom, padding-left, padding-right, and it stayed centered. Height is given and `:align-items        :center :justify-content    :center` is used, so these extra paddings are redundant. 

Also, there is `horizontal-padding` given, why is there also padding-left and right?

Reply:
@J-Son89 Yeah seperate PR. This PR just fixes vertical alignment for size 24

Reply:
sounds good, thanks! 🙌